benches/benches.rs

```diff
@@ -0,0 +1,84 @@
+#![feature(bench_black_box)]
```

Comment:
Not sure we need these benchmarks 😀

Reply:
They've already paid for themselves (by catching perf regression from prev version) - e.g. inline attribute on custom so the custom construction is as fast as before. Gives people peace of mind that they've not changed the perf going forward. Before I wrote them I was thinking this was overkill but am now totally sold on them.